src/librustc_ast/Cargo.toml

```diff
@@ -19,3 +19,4 @@ rustc_index = { path = "../librustc_index" }
 rustc_lexer = { path = "../librustc_lexer" }
 rustc_macros = { path = "../librustc_macros" }
 smallvec = { version = "1.0", features = ["union", "may_dangle"] }
+rustc_target = { path = "../librustc_target" }
```

Comment:
`rustc_ast` and `rustc_target` were only recently decoupled (https://github.com/rust-lang/rust/pull/65884) and it would be great to keep them decoupled.
Not sure why `InlineAsmTemplatePiece` and `InlineAsmOptions` are in `rustc_target`, they don't look target-dependent.

Reply:
OK, I'll move them to librustc_ast.